data-prepper-core/src/main/java/com/amazon/dataprepper/parser/model/DataPrepperConfiguration.java

```diff
@@ -71,6 +78,15 @@ public String getPrivateKeyPassword() {
         return privateKeyPassword;
     }
 
+    public List<MeterRegistryType> getMetricsRegistries() {
+        return metricsRegistries;
+    }
+
+    public boolean isMeterRegistryTypeConfigured(final MeterRegistryType meterRegistryType) {
```

Comment:
I see this method is used a bit further down to check if a Prometheus registry exists, then `getPrometheusMeterRegistryFromRegistries` is called to do a similar search (but return the object instead of a boolean).

Could this method be changed to something like `Optional<MeterRegistry> getRegisteredMeterByType(...)`? Callers could then `result.isPresent()` to get the same bool functionality.